lab_3_keywords_textrank/main.py

```diff
@@ -120,7 +133,13 @@ def _learn_indices(self, tokens: tuple[str, ...]) -> None:
             tokens : tuple[str, ...]
                 sequence of string tokens
         """
-        pass
+        id_min = 1000
+        for word in tokens:
+            if word in self._word2id:
+                continue
+            self._word2id[word] = id_min
+            self._id2word[id_min] = word
+            id_min += 1  # not working
```

Comment:
why it is not working?

Reply:
it works, but the tokens' numbers are not consequent (for example a phrase "five black cats" can be tokenized as "1001, 1005, 1007"). I'm not sure if it's okay, but it works, not really well

Reply:
I think it is ok